apps/fabric-website/package.json

```diff
@@ -71,8 +71,8 @@
     "color-functions": "1.1.0",
     "json-loader": "^0.5.4",
     "load-themed-styles": "^1.1.0",
-    "office-ui-fabric-core": ">=7.1.2 <8.0.0",
-    "office-ui-fabric-react": ">=4.35.2 <5.0.0",
+    "office-ui-fabric-core": "^7.2.0",
+    "office-ui-fabric-react": "^4.35.2",
```

Comment:
you should not be messing with the fabric-react version. Publishing scripts auto adjust this number...

Technically, also, the fabric-website should not have its own dependency on core. It can pull things from the oufr dependency.